src/components/playback/playbackmanager.js

```diff
@@ -3860,7 +3860,6 @@ define(['events', 'datetime', 'appSettings', 'itemHelper', 'pluginManager', 'pla
                 "SetMaxStreamingBitrate",
                 "DisplayContent",
                 "GoToSearch",
-                "DisplayMessage",
```

Comment:
why?

Reply:
We really need to document the client remote control feature set, right now everything is just getting passed around blindly.

Reply:
These are the features of the player passed to the server during local playback. (I think so)
And I think I would never send me a message to the same client...
For the KODI addon it is listed here https://github.com/jellyfin/jellyfin-kodi/blob/f44efb8e4cb99de6371eb6be21d9a6ddd2db180d/jellyfin_kodi/monitor.py#L305

I have just seen that I might have to remove three other features:
![image](https://user-images.githubusercontent.com/14938497/69434396-d5d69f00-0d3d-11ea-8b76-c9236ebd577b.png)

Maybe I'm able to hide the navigation section and both text sections completely if I can check on the now playing screen if it is local playback.
Then we could keep the features for the local player as it is.

Reply:
Now it should be better